pkg/psmdb/statefulset.go

```diff
@@ -354,6 +356,52 @@ func StatefulSpec(ctx context.Context, cr *api.PerconaServerMongoDB, replset *ap
 		}
 	}
 
+	if cr.CompareVersion("1.22.0") >= 0 && cr.Spec.Backup.Enabled {
+		cas := collectStorageCABundles(cr)
+
+		if len(cas) == 1 {
```

Comment:
On another thought, I wonder if we really need to check the length of the provided CAs..

What if the user-provided CAs always go to the `ca-bundle-in` volume (even if 1 CA is specified), and the init container append everything to the `ca-bundle.crt` in the emptydir volume like it does now.. That way we don't have to check the length here and can simplify the code, wdyt?